lib/web/desktop.go

```diff
@@ -34,6 +37,27 @@ import (
 	"github.com/gravitational/teleport/lib/utils"
 )
 
+// TerminalRequest describes a request to create a web-based terminal
+// to a remote SSH server.
+type DesktopAccessQueryParams struct {
+	// Login is Windows username to connect as.
+	Login string `json:"login"`
+
+	// Screen is the initial screen size.
+	Screen struct {
+		W int `json:"w"`
+		H int `json:"h"`
+	} `json:"screen"`
+}
+
+// GET /webapi/sites/:site/desktops/:desktopName/connect?access_token=bearer_token&params=<urlencoded json-structure>
+//
+// Due to the nature of websocket we can't POST parameters as is, so we have
```

Comment:
This isn't even a `POST` endpoint, so I don't think we should be talking about POSTing paramaters.
```suggestion
// Due to the nature of websocket we can't provide a request body, so we have
// to encode connection information in query parameters.
```